src/Fridge.java

```diff
@@ -0,0 +1,57 @@
+import components.map.Map;
+import components.map.Map1L;
+
+public class Fridge {
+    private boolean isOpen;
+    private static Map<String, Integer> fridgeItems;
+    private Map<String, Integer> freezerItems;
+
+    public Fridge() {
+        this.isOpen = false;
+        this.fridgeItems = new Map1L<>();
+        this.freezerItems = new Map1L<>();
+    }
+
+    public void addOneItem(String item, boolean isOpen,
+            Map<String, Integer> location) {
+        if (!isOpen) {
+            if (location.hasKey(item)) {
+                location.replaceValue(item, location.value(item) + 1);
+            } else {
+                location.add(item, 1);
+            }
+        }
+    }
+
+    public void removeOneItem(String item, boolean isOpen,
+            Map<String, Integer> location) {
+        if (!isOpen) {
+            location.remove(item);
+        }
+    }
+
+    public boolean inFridge(String item) {
```

Comment:
Great names! I think the typical naming convention for methods that return booleans is something like the following:

```suggestion
    public boolean isInFridge(String item) {
```